web_src/js/components/DashboardRepoList.vue

```diff
@@ -255,6 +256,40 @@ const sfc = {
     $(el).find('.dropdown').dropdown();
     nextTick(() => {
       this.$refs.search.focus();
+      this.$refs.search.addEventListener('keydown', (e) => {
+        const previousPageLink = this.$refs.previousPageLink;
+        const nextPageLink = this.$refs.nextPageLink;
+        switch (e.key) {
+          case 'Enter':
+            this.$refs[`activeIndex${this.activeIndex}`][0].click();
+            break;
+          case 'ArrowUp':
+            if (this.activeIndex > 0) {
+              this.activeIndex--;
+            } else if (previousPageLink && !previousPageLink.classList.contains('disabled')) {
+              previousPageLink.click();
```

Comment:
Using Up/Down to change page doesn't seem right.

For example:

* Suppose a user is seeing a list [a,b,c] on page 10, the first a is active, then Up is pressed, they sees the page 9, but the first item of page 9 is active? Press Up again, then they goes to page 8 immediately? Normally I guess users expects to see the active item is the last one in this case.
* If the network is laggy, after the first Up press, if the user presses Up again, they skips page 9 and sees page 8 directly?

I think these are unexpected behaviors.

Reply:
Maybe solved?!

Reply:
> Using Up/Down to change page doesn't seem right.

How is it resolved for these 2 cases?

Reply:
You are right, if you start from the first stroke, you will jump directly to the previous page. Do you want to select the last stroke on the previous page? Don't you want to choose the first stroke on the previous page?
If so, I can add a line `this.activeIndex = this.searchLimit - 1;` in the case of the up key

Reply:
I added it.